### PR TITLE
fix: add key and verifier registration to guacone cli

### PR DESCRIPTION
**Pull request by pxp928**
Adding the missing key and sigstore verifier registration to the guacone CLI to have the ability to verify DSSE and create identity nodes.

Address Issue #304

### CODE REVIEW

File: cmd/guacone/cmd/files.go [hunk 1]
```diff
@@ -63,13 +75,42 @@ var exampleCmd = &cobra.Command{
 			viper.GetString("gdbpass"),
 			viper.GetString("gdbaddr"),
 			viper.GetString("realm"),
+			viper.GetString("keyPath"),
+			viper.GetString("keyID"),
```
Comment (lumjjb) [inline cmd/guacone/cmd/files.go:79]: can we maybe add a prefix for some of these arguments, maybe `verifier-keyPath` and `verifier-keyID`

File: cmd/guacone/cmd/files.go [hunk 2]
```diff
@@ -63,13 +75,42 @@ var exampleCmd = &cobra.Command{
...
+			}
+			err = key.Store(ctx, opts.keyID, keyRaw, inmemory.Type())
+			if err != nil {
+				logger.Errorf("error: %v", err)
+				os.Exit(1)
+			}
+		}
+
+		// Register Verifier
+		sigstore := sigstore_verifier.NewSigstoreVerifier()
```
Comment (lumjjb) [inline cmd/guacone/cmd/files.go:108]: nit: maybe renaming this to `SigstoreAndKeyVerifier` for now just for accuracy.

**Review by lumjjb — COMMENTED** [review]
minor nits, else LGTM

**Review by lumjjb — APPROVED** [review]
lgtm

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Pull request merged**